packages/styles/globals.import.styl

```diff
@@ -117,6 +117,11 @@ a
     top auto
     width 100%
```

Comment:
Looks like it is still referenced in a coffee file:
`git grep toast-top-right-custom` results in:
packages/controllers/toastr.coffee:  positionClass: 'toast-top-right-custom'
packages/styles/globals.import.styl:.toast-top-right-custom